src/libraries/System.Runtime/tests/System.IO.Tests/TextWriter/TextWriterTests.cs

```diff
@@ -685,6 +685,186 @@ public async Task FlushAsync_Precanceled()
+
+            oracle.WriteLine(47u);
+            broadcasting.WriteLine(47u);
+
+            oracle.WriteLine(48ul);
+            broadcasting.WriteLine(48ul);
+
+            oracle.WriteLine("Thursday".AsSpan());
+            broadcasting.WriteLine("Thursday".AsSpan());
+
+            oracle.WriteLine(" {0} ", "Friday");
+            broadcasting.WriteLine(" {0} ", "Friday");
+
+            oracle.WriteLine(" {0}{1} ", "Saturday", "Sunday");
+            broadcasting.WriteLine(" {0}{1} ", "Saturday", "Sunday");
+
+            oracle.WriteLine(" {0} {1}  {2}", TimeSpan.FromSeconds(1), TimeSpan.FromMinutes(2), TimeSpan.FromDays(3));
+            broadcasting.WriteLine(" {0} {1}  {2}", TimeSpan.FromSeconds(1), TimeSpan.FromMinutes(2), TimeSpan.FromDays(3));
+
+            oracle.WriteLine(" {0} {1}  {2}    {3}", (Int128)4, (UInt128)5, (nint)6, (nuint)7);
+            broadcasting.WriteLine(" {0} {1}  {2}    {3}", (Int128)4, (UInt128)5, (nint)6, (nuint)7);
+
+            await oracle.WriteAsync('a');
+            await broadcasting.WriteAsync('a');
+
+            await oracle.WriteAsync((char[])null);
+            await broadcasting.WriteAsync((char[])null);
+            await oracle.WriteAsync(new char[] { 'b', 'c' });
+            await broadcasting.WriteAsync(new char[] { 'b', 'c' });
+
+            await oracle.WriteAsync((string)null);
+            await broadcasting.WriteAsync((string)null);
+            await oracle.WriteAsync("Tuesday");
+            await broadcasting.WriteAsync("Tuesday");
+
+            await oracle.WriteAsync((StringBuilder)null);
+            await broadcasting.WriteAsync((StringBuilder)null);
+            await oracle.WriteAsync(new StringBuilder("Wednesday"));
+            await broadcasting.WriteAsync(new StringBuilder("Wednesday"));
+
+            await oracle.WriteLineAsync();
+            await broadcasting.WriteLineAsync();
+
+            await oracle.WriteLineAsync('a');
+            await broadcasting.WriteLineAsync('a');
+
+            await oracle.WriteLineAsync((char[])null);
+            await broadcasting.WriteLineAsync((char[])null);
+            await oracle.WriteLineAsync(new char[] { 'b', 'c' });
+            await broadcasting.WriteLineAsync(new char[] { 'b', 'c' });
+
+            await oracle.WriteLineAsync((string)null);
+            await broadcasting.WriteLineAsync((string)null);
+            await oracle.WriteLineAsync("Tuesday");
+            await broadcasting.WriteLineAsync("Tuesday");
+
+            await oracle.WriteLineAsync((StringBuilder)null);
+            await broadcasting.WriteLineAsync((StringBuilder)null);
+            await oracle.WriteLineAsync(new StringBuilder("Wednesday"));
+            await broadcasting.WriteLineAsync(new StringBuilder("Wednesday"));
+
+            string expected = oracle.ToString();
+            Assert.Equal(expected, sw1.ToString());
+            Assert.Equal(expected, sw2.ToString());
+        }
+
```

Comment:
Could you please add a test that ensures that `BroadcastingTextWriter` overrides all virtual methods of `TextWriter`? I would like to make sure that we never miss it when adding new `virtual` methods to `TextWrtier`.